internal/pkg/cli/env_init.go

```diff
@@ -477,6 +479,7 @@ func (o *initEnvOpts) deployEnv(app *config.Application) error {
 		AdditionalTags:           app.Tags,
 		AdjustVPCConfig:          o.adjustVPCConfig(),
 		ImportVPCConfig:          o.importVPCConfig(),
+		Version:                  envVersion,
```

Comment:
can we use `deploy.LatestEnvTemplateVersion`?

Reply:
oh i didn't realize this

Reply:
actually I take this back, I like what you have with pinning `env init` to a specific version. 
So that we can work on a new template without immediately shipping it.

But instead, can we move the const to be in the deploy package: `deploy.EnvTemplateVersion100` maybe?